typescript/cli/src/status/message.ts

```diff
@@ -94,7 +94,11 @@ export async function checkMessageStatus({
     const merkleAddress = chainAddresses[origin].merkleTreeHook;
     stubMerkleTreeConfig(relayer, origin, hookAddress, merkleAddress);
 
-    deliveredTx = await relayer.relayMessage(dispatchedReceipt);
+    deliveredTx = await relayer.relayMessage(
+      dispatchedReceipt,
+      undefined,
```

Comment:
I'm surprised setting this to `undefined` works, given the arg has type `number`

Reply:
haha same, but i'm just copying what we did in a different part of the code